packages/components/datetime/src/RelativeDateTime/README.mdx

```diff
@@ -17,21 +17,19 @@ Provides a relative date string for resolutions ranging from seconds up through
 
 ### Examples
 
-If we consider today as of August 18th of 2021 and we pass the following values to the `<RelativeDateTime />` component
+We can pass a date as either an ISO 8601 string, a Date object, or a UNIX Epoch time.
```

Comment:
We could link to definitions here maybe? Or it's an overkill?

Reply:
I thought about it but decided it was too many external links to Wikipedia 😅 I think for now it's ok to ask people to google it if they're unsure what it is.